tempesta_fw/http_sess.c

```diff
@@ -402,7 +402,7 @@ tfw_http_sticky_verify(TfwHttpReq *req, TfwStr *value, StickyVal *sv)
 	}
 
 	TFW_STR_FOR_EACH_CHUNK(c, value, end) {
-		for (p = c->ptr; p < (unsigned char *)c->ptr + c->len; ++p) {
+		for (p = c->data; p < (unsigned char *)c->data + c->len; ++p) {
```

Comment:
If `c->data` is used just as a pointer, is there the need to cast it from `(char *)` to `(unsigned char *)`?